keda/templates/keda-vpa.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.vpa.keda.enabled }}
```

Comment:
I think I would use verticalPodAutoscaler instead of vpa

Reply:
Done

Reply:
perfect